phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/installations/dto/InstallationDTO.java

```diff
@@ -61,16 +65,19 @@ private InstallationDTO(Builder builder) {
         node = builder.node;
     }
 
+    /**
+     * type of the installation
+     */
     public enum InstallationType {
-        APPS, WINE;
+        APPS, ENGINES;
 
         @Override
         public String toString() {
             switch (this) {
```

Comment:
What will happen if someone puts a typo in the display string? Than it might suddenly not be translated anymore etc.